src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/ListViewModel.cs

```diff
@@ -248,6 +266,31 @@ private void InitializeItemsTask(CancellationToken ct)
         }
     }
 
+    public void UpdateGroupsIfNeeded()
+    {
+        Groups.Clear();
+        if (HasGrouping)
+        {
+            try
+            {
+                var groupedItems = FilteredItems
+                    .Where(item => !string.IsNullOrEmpty(item.Section))
```

Comment:
This might be more of an API discussion - how should we treat a list of items where they don't all have a `Section` set?

* Do we just ignore them? _this feels bad_
* Do we create a first "section" that doesn't have a title to it? 
  * Do we create that section in the same ordering as all the other sections?
  * Do we have some placeholder "(no section)" text? _this feels bad_

Reply:
yeah, I think having all of these in an unlabeled group most sense